sites/default/statement.inc.php

```diff
@@ -102,278 +463,280 @@
 //  [Your billing dept phone]
 
 function create_statement($stmt) {
```

Comment:
Just making sure you meant to make these changes in the orginal statement also.

The one thing that seems off is that the $billing_phone is getting replaced by $GLOBALS['billing_phone_number'], which is new behavior. Probably only makes sense to do this if there is a $GLOBALS['billing_phone_number'] set in globals (thus will allow practices/groups to set a global billing phone number if they want or instead grab it from the pertinent facility as I think was previously done).